launch-sublime-platform.sh

```diff
@@ -1,18 +1,53 @@
 #!/bin/bash
 
-if [ -z "$auto_updates" ]; then
+. ./utils.sh
+
+if [ -z "$interactive" ]; then
+    interactive="true"
+fi
+
+if [ "$interactive" != "true" ] && [ -z "$auto_updates" ]; then
     auto_updates=true
 fi
```

Comment:
Same as `install-and-launch.sh` in case users want to call this script directly.